<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parityBits</class>
 <widget class="QWidget" name="parityBits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>parityBits</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelBinary">
         <property name="text">
          <string>Введите бинарное сообещние</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditBinary"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="calculateParityLabel">
         <property name="midLineWidth">
          <number>-1</number>
         </property>
         <property name="text">
          <string>Паритет для текущего сообщения</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resOfCalcParityLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="parityRadBtn">
         <property name="text">
          <string>Контроль по паритету</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="VerHorParityRadBtn">
         <property name="text">
          <string>Вертикальный/горизонтальный паритет</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="imitateErrorsCheckBox">
         <property name="text">
          <string>Имитация ошибки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkButton">
         <property name="text">
          <string>Проверить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
